js/query_parser.js

```diff
@@ -0,0 +1,11 @@
+var urlParams;
+(window.onpopstate = function () {
+  var match
```

Comment:
Is this your work? If not you need to at least provide a source. It _looks_ like you stole this from a **question** on StackOverflow: https://stackoverflow.com/q/23648945

Reply:
Was copied from a personal project, so probably stolen....